packages/create-catalyst/bin/index.cjs

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+const semver = require('semver');
+
+const catalystRequiredVersion = '>=20';
```

Comment:
🍹 Might say `catalystRequiredNodeVersion` as this sounds like it's referring to the catalyst version itself.